examples/datasource-http-backend/README.md

```diff
@@ -45,14 +43,20 @@ $ mage -v
 
 ### Example server
 
+mockserver is included to docker-compose file. You can run or run server separetaly 
+
 ```bash
-$ mage server
-$ ./cmd/server/server :10000
-2022/10/28 15:43:16 listening on :10000
+cd server
+docker-compose up
```

Comment:
Since this section talks only about the server this command should be `docker compose up mockserver`